node_modules/oae-content/tests/test-content.js

```diff
@@ -2963,15 +2963,22 @@ describe('Content', function() {
                                     RestAPI.Content.updateMembers(contexts['nicolaas'].restContext, contentObj.id, permissions, function(err) {
                                         assert.equal(err.code, 400);
 
-                                        // Sanity check
+                                        // The value `editor` is only valid on collabdocs
                                         permissions = {};
-                                        permissions[contexts['simon'].user.id] = 'manager';
+                                        permissions[contexts['simon'].user.id] = 'editor';
                                         RestAPI.Content.updateMembers(contexts['nicolaas'].restContext, contentObj.id, permissions, function(err) {
-                                            assert.ok(!err);
-                                            RestAPI.Content.updateContent(contexts['simon'].restContext, contentObj.id, {'displayName': 'Sweet stuff'}, function(err, updatedContentObj) {
+                                            assert.equal(err.code, 400);
+
+                                            // Sanity check
+                                            permissions = {};
+                                            permissions[contexts['simon'].user.id] = 'manager';
+                                            RestAPI.Content.updateMembers(contexts['nicolaas'].restContext, contentObj.id, permissions, function(err) {
                                                 assert.ok(!err);
-                                                assert.equal(updatedContentObj.displayName, 'Sweet stuff');
-                                                callback();
+                                                RestAPI.Content.updateContent(contexts['simon'].restContext, contentObj.id, {'displayName': 'Sweet stuff'}, function(err, updatedContentObj) {
+                                                    assert.ok(!err);
+                                                    assert.equal(updatedContentObj.displayName, 'Sweet stuff');
+                                                    callback();
+                                                });
```

Comment:
Can you also add a few tests that editors can't make other people a manager, delete content items, etc..